bioframe/io/fileops.py

```diff
@@ -505,35 +505,11 @@ def to_bigwig(df, chromsizes, outpath, value_field=None, path_to_binary=None):
         is to use the fourth column.
     path_to_binary : str, optional
         Provide system path to the bedGraphToBigWig binary.
+    engine : {'kenttools', 'pybigtools'}, optional
```

Comment:
I'd go with `ucsc` and `bigtools` as names.

Reply:
done!